components/button/index.en-US.md

```diff
@@ -41,7 +41,7 @@ Different button styles can be generated by setting Button properties. The recom
 | icon | Set the icon component of button | ReactNode | - |  |
 | loading | Set the loading status of button | boolean \| { delay: number } | false |  |
 | shape | Can be set button shape | `circle` \| `round` | - |  |
-| size | Set the size of button | `large` \| `middle` \| `small` | - |  |
+| size | Set the size of button | `large` \| `middle` \| `small` | `middle` |  |
```

Comment:
```suggestion
| size | Set the size of button | `large` \| `default` \| `small` | `default` |  |
```

Reply:
These are already in use:

https://github.com/ant-design/ant-design/blob/0658d3278539e0d687c6f1b3dd83a3daf5f7ab91/components/config-provider/SizeContext.tsx#L3

This just brings the docs up to date.